scripts/config-doc-command.js

```diff
@@ -106,19 +111,20 @@ try {
   const hasTag = doc.includes(TAGS.CONFIG.START)
   const hasUsageTag = doc.includes(TAGS.USAGE.START)
 
-  if (params.length) {
+  if (params?.length) {
     let newDoc = hasTag ? addDescriptions(doc) : doc
     newDoc = hasUsageTag ? addUsageDescriptions(newDoc) : newDoc
 
     if (!hasTag) {
       console.error('WARNING: did not find config description section', configDoc)
     }
 
-    if (usage.length && !hasUsageTag) {
+    if (usage?.length && !hasUsageTag) {
```

Comment:
This isn't supported in all the versions of node we have to run in.

Reply:
reverted.